Automattic/newspack-plugin
# Reader Activation: customizable verification email

### All Submissions:






### Changes proposed in this Pull Request:



Two closely related changes:

- Redesign of the verification email
- Enabling customization of the verification email (sent when a reader registers)

Default email design:

<img width="630" alt="image" src="https://user-images.githubusercontent.com/7383192/186357254-d9f87a62-e683-4fe1-863c-5a4b07d6d097.png">

### How to test the changes in this Pull Request:

1. Since this changeset contains a refactor of the Reader Revenue email, test the post-Stripe-donation email flow:
    1. Make a donation with Stripe configured as the RR platform, observe the email is sent
    2. Edit the post-donation email, observe it's sent as the updated version
3. With Reader Activation enabled, register as a new reader
4. Observe an email is sent, with the updated design 
5. Visit Engagement wizard, "Reader Activation" screen, "Emails" section
6. Edit the "Verification" email
7. In a new session, register as a new reader. Observe the updated email is sent

### Other information:
